playground/src/markdown/Explainer.md

```diff
@@ -0,0 +1,81 @@
+# What the f*ck? A short explainer
+
+This is a Builder's Playground for the Miden Virtual Machine. You can create any program in Miden Assembly run, debug, prove and verify it. Or just check out our examples. All in the browser.
+
+## What is the Miden VM?
+The Miden VM is a zero-knowledge virtual machine. A regular virtual machine consumes an initial state and a program and produces a final state.
+
+![](https://i.imgur.com/y3yYt2R.png)
+
+The Miden VM works like a regular virtual machine. Only that you can provide secret inputs (Witness) and together with the final state the Miden VM creates a proof for it. A zero-knowledge virtual machines proves that a given set of inputs and program code results in the final state.  
+
+![](https://i.imgur.com/t517366.png)
+
+## Inputs of the Miden VM
+External inputs can be provided to Miden VM in the Playground in two ways:
+
+![](https://i.imgur.com/n0LZpvU.png)
```

Comment:
This image isn't great

![image](https://user-images.githubusercontent.com/310139/225678809-4491564d-f7fa-43bf-a9e2-3706d6a7541e.png)

The gradient on the left side doesn't seem to have any purpose and the labels are not really aligned. Perhaps remove the image and use inline code that is native to common markdown?

```suggestion
```json
{
  "stack_init": ["0"],
  "advice_tape": ["0"]
}
```

Even better if we can add syntax highlight